requirements.txt

```diff
@@ -1,4 +1,6 @@
+aiohttp==3.5.4
 cryptography==2.7
 PyMySQL==0.9.3
 pytest==4.6.3
+Requests==2.22.0
```

Comment:
Why do we need requests?

Reply:
It's required for `google.auth.transport.requests`, which is used for refreshing credentials if they're not valid.

Reply:
Just to clarify - why is requests not a dependency of `google.auth.transport.requests` then?